commands/operator-sdk/cmd/new.go

```diff
@@ -143,50 +248,48 @@ func repoPath() string {
 }
 
 func verifyFlags() {
-	if len(apiVersion) == 0 {
-		cmdError.ExitWithError(cmdError.ExitBadArgs, errors.New("--api-version must not have empty value"))
-	}
-	if len(kind) == 0 {
-		cmdError.ExitWithError(cmdError.ExitBadArgs, errors.New("--kind must not have empty value"))
-	}
 	if operatorType != goOperatorType && operatorType != ansibleOperatorType {
-		cmdError.ExitWithError(cmdError.ExitBadArgs, errors.New("--type can only be `go` or `ansible`"))
+		log.Fatal("--type can only be `go` or `ansible`")
 	}
 	if operatorType != ansibleOperatorType && generatePlaybook {
-		cmdError.ExitWithError(cmdError.ExitBadArgs, errors.New("--generate-playbook can only be used with --type `ansible`"))
-	}
-	kindFirstLetter := string(kind[0])
-	if kindFirstLetter != strings.ToUpper(kindFirstLetter) {
-		cmdError.ExitWithError(cmdError.ExitBadArgs, errors.New("--kind must start with an uppercase letter"))
+		log.Fatal("--generate-playbook can only be used with --type `ansible`")
 	}
-	if strings.Count(apiVersion, "/") != 1 {
-		cmdError.ExitWithError(cmdError.ExitBadArgs, fmt.Errorf("api-version has wrong format (%v); format must be $GROUP_NAME/$VERSION (e.g app.example.com/v1alpha1)", apiVersion))
+	if operatorType == goOperatorType && (len(apiVersion) != 0 || len(kind) != 0) {
+		log.Fatal("go type operator does not use --api-version or --kind. Please see generate command after running new.")
```

Comment:
Nit:
Change `Please see generate command after running new`
to `Please see "operator-sdk add" command after running new`